app/org/sagebionetworks/bridge/models/accounts/StudyParticipant.java

```diff
@@ -172,7 +178,8 @@ public boolean equals(Object obj) {
                 && Objects.equals(id, other.id) && Objects.equals(languages, other.languages)
                 && Objects.equals(lastName, other.lastName) && Objects.equals(notifyByEmail, other.notifyByEmail)
                 && Objects.equals(password, other.password) && Objects.equals(roles, other.roles)
-                && Objects.equals(sharingScope, other.sharingScope) && Objects.equals(status, other.status);
+                && Objects.equals(sharingScope, other.sharingScope) && Objects.equals(status, other.status)
+                && Objects.equals(timeZone, other.timeZone);
```

Comment:
This is a comparison test for the study participant and is mostly used for tests, so we want to know if the participant object has a different time zone associated with it. Doesn't compare two time stamps.